<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialup_conf</class>
 <widget class="QDialog" name="dialup_conf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color: rgb(135, 135, 104);}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>182</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>4</y>
     <width>310</width>
     <height>210</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed{background-color: rgb(255, 186, 181);border-width: 5px;border-color:Magenta;border-style: solid;}</string>
   </property>
   <property name="title">
    <string>Dial-up Settings</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>18</y>
      <width>100</width>
      <height>155</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnAdd_new">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>191</width>
      <height>155</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <tabstops>
  <tabstop>btnAdd_new</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialup_conf</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialup_conf</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
